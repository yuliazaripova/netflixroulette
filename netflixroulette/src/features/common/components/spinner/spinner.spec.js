import * as React from 'react';

import Spinner from './spinner';

describe("Spinner component", () => {
  it("should render Spinner component", () => {
    const component = shallow(<Spinner />);
    expect(component).toMatchSnapshot();
  })
})