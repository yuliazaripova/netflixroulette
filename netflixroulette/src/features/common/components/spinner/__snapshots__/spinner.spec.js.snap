// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should render Spinner component 1`] = `
<div
  className="lds-css"
>
  <div
    className="lds-double-ring"
  >
    <div />
    <div />
  </div>
</div>
`;
